components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/util/FrameworkUtils.java

```diff
@@ -3000,14 +3000,38 @@ public static String preprocessUsername(String username, ServiceProvider service
         return username;
     }
 
+    /**
+     * Retrieves the ResolvedUserResult from the respective login resolver if it is enabled in the management console.
+     *
+     * @param loginIdentifier login identifier for multi attribute login
+     * @param tenantDomain    The tenant domain of the user.
+     * @return resolvedUserResult instance with a SUCCESS status if the login resolver is enabled. Otherwise, returns
+     * resolvedUserResult with FAIL status.
+     */
+    public static org.wso2.carbon.identity.login.resolver.mgt.ResolvedUserResult processLoginResolverIdentification(
```

Comment:
Let's import the org.wso2.carbon.identity.login.resolver.mgt.ResolvedUserResult to the java class

Reply:
There are two ResolvedUserResult classes from different packages hence we only can import one. The other one should be used with the fully qualified package name :upside_down_face:.

Reply:
This should be fine